updater/lib/dependabot/updater/operations/create_security_update_pull_request.rb

```diff
@@ -69,8 +80,11 @@ def check_and_create_pr_with_error_handling(dependency)
             error_type: "inconsistent_registry_response",
```

Comment:
[I think the error is surfaced here](https://github.com/dependabot/dependabot-core/blob/6c54922e889cb53c96f3d132d14e9f185b2b1765/npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater.rb#L48-L56).

can we detect the actual transitive failure mode and add a specific error class for that scenario? That way we are not swallowing up all errors.

I would expect something like this:

```
raise Dependabot::SecurityUpdateFailureDueToTransitiveDependency.new(...
...
```

That way, we can ensure we are actually separating out that error category.

Reply:
@abdulapopoola , im working on this